### PR TITLE
Update links and README

### PR DESCRIPTION
**Pull request by patriknw**

### CODE REVIEW

File: src/main/g8/pom.xml [hunk 1]
```diff
@@ -6,20 +6,16 @@
...
-    <properties>
-        <akkaVersion>2.5.1</akkaVersion>
-    </properties>
-
     <dependencies>
         <dependency>
             <groupId>com.typesafe.akka</groupId>
             <artifactId>akka-actor_2.12</artifactId>
-            <version>${akkaVersion}</version>
+            <version>2.5.1</version>
```
Comment (patriknw) [inline src/main/g8/pom.xml]: there is still some problem with using such maven property when using g8, so I simply removed it for now

File: README.md [hunk 2]
```diff
@@ -1,14 +1,50 @@
...
+
+It can also be used as a [Giter8][g8] template for Akka with Java.
+
+Prerequisites:
+- JDK 8
+- [sbt][sbt] 0.13.13 or higher ([download here][sbt_download])
+
+Open a console and run the following command to apply this template:
+ ```
+sbt -Dsbt.version=0.13.15 new https://github.com/akka/akka-java-seed.g8
```
Comment (2m) [inline README.md]: Shorter version would be:

    sbt new akka/akka-java-seed.g8

File: README.md [hunk 3]
```diff
@@ -1,14 +1,50 @@
...
+ ```
+
+This template will prompt for the following parameters. Press `Enter` if the default values suit you:
+- `name`: Becomes the name of the project.
```
Comment (2m) [inline README.md]: Same rewording as in scala seed:

> This template will prompt for the name of the project. Press Enter if the default value suits you.

File: README.md [hunk 4]
```diff
@@ -1,14 +1,50 @@
...
+
+Once inside the project folder, to run Hello World with:
```
Comment (2m) [inline README.md]: Reword to:

> Once inside the project folder, run the application with:

File: README.md [hunk 5]
```diff
@@ -1,14 +1,50 @@
...
-License
--------
-Written in 2014-2016 by Lightbend, Inc.
-To the extent possible under law, the author(s) have dedicated all copyright and related and neighboring rights to
-this template to the public domain worldwide. This template is distributed without any warranty.
-See <http://creativecommons.org/publicdomain/zero/1.0/>.
+This template also provides build descriptors for maven and gradle. You can use any of the following commands to run 
+the application:
+```
+mvn exec:exec
```
Comment (2m) [inline README.md]: We need to explicitly call `compile` before trying to run. So full command would be:

    mvn compile exec:exec

Comment (patriknw) [inline README.md]: ah, so maven doesn't understand that dependency?

Comment (patriknw) [inline README.md]: ok, seems true, updating

Comment (2m) [inline README.md]: Unfortunately no.

**Comment by patriknw** [comment]
updated

**Comment by patriknw** [comment]
crashed the jvm, restarting

**Comment by 2m** [comment]
Great. We can merge this. (I do not have permissions...)

**Comment by patriknw** [comment]
@2m I have update the permissions

**Pull request merged**